src/levanter/infra/ray_tpu.py

```diff
@@ -25,8 +25,12 @@
 
 # CF https://gist.github.com/allenwang28/e3400b9e9212b50aa1cda55ebeccea60
 
+from levanter.ray_tpu.src.ray_tpu import RayTpuManager
+
 logger = logging.getLogger("ray")
 
+_tpu_manager = RayTpuManager()
```

Comment:
i think i'd rather not initialize a global here unless  it's a pretty simple state